model/network.yaml

```diff
@@ -25,6 +25,9 @@ groups:
           [OSI Transport Layer](https://osi-model.com/transport-layer/) or
           [Inter-process Communication method](https://en.wikipedia.org/wiki/Inter-process_communication).
           The value SHOULD be normalized to lowercase.
+          Consider always setting the transport when setting a port number, since
```

Comment:
Maybe we can set the requirement level to something `conditionally_required: if instrumented technology supports different transports and value cannot be inferred from other attributes such as `network.protocol.name`` - this would also eliminate the concern for ports and require instrumentations to populate it whenever it's helpful.

Also, it will make brief more brief.

Reply:
> Also, it will keep brief more brief.

ya, I think we have a (general) brief problem, tracking here: #335

Reply:
opened #353 to track https://github.com/open-telemetry/semantic-conventions/pull/289#discussion_r1336505431, resolving this conversation